src/main/app/case/case-api-client.ts

```diff
@@ -128,7 +128,10 @@ export class CaseApiClient {
     } catch (err) {
       if (
         retries < this.maxRetries &&
-        (err?.response.status === 409 || err?.response.status === 422 || err?.response.status === 504)
+        (err?.response.status === 409 ||
```

Comment:
i think at this point we can change this to
[409, ...].includes(err?.response.status)